superset-frontend/src/explore/components/ExploreChartPanel.jsx

```diff
@@ -113,13 +126,12 @@ class ExploreChartPanel extends React.PureComponent {
         chart={this.props.chart}
       />
     );
+
     return (
-      <div className="chart-container">
-        <Panel style={{ height: this.props.height }}>
-          <Panel.Heading>{header}</Panel.Heading>
-          <Panel.Body>{this.renderChart()}</Panel.Body>
-        </Panel>
-      </div>
+      <Styles className="chart-container">
+        <div>{header}</div>
+        <div>{this.renderChart()}</div>
+      </Styles>
```

Comment:
@rusackas by not using the `Panel` component, you also missed some CSS styles for `.panel-header` and `panel-body`.

### After

![image](https://user-images.githubusercontent.com/335541/89149455-5b3b6180-d511-11ea-855f-52f41888e472.png)

### Before

![image](https://user-images.githubusercontent.com/335541/89149475-6db59b00-d511-11ea-8aca-e87147384a64.png)

Reply:
Just pushed a [fix](https://github.com/apache/incubator-superset/pull/10505). I'm using the raw boostrap classes for now. Let's refactor it into Antd components or a more self-contained styled component in the future.